pixi/src/utils/texts.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const htmlEntityMapping = {
+  // we will not use the html inside attributes
+  // so here we can use the normal characters...
+  '&quot;': '"',
+  '&#39;': "'",
+  // since the following caracters can not be used unescaped
+  // we use unicode replacements...
+  '&gt;': '\uFE65', // small greater than sign
```

Comment:
I'm confused, why couldn't we replace `&gt` with `>`?

Reply:
see comment above